etc/kayobe/overcloud-dib.yml

```diff
@@ -10,6 +10,10 @@
 # is False. This will change in a future release.
 #overcloud_dib_build_host_images:
 
+# List of additional host packages to install.
+overcloud_dib_host_packages_extra:
+  - "{% if 'ubuntu-minimal' in overcloud_dib_host_images | map(attribute='elements') | flatten | list %}debootstrap{% endif %}"
```

Comment:
This requires a backport of https://review.opendev.org/c/openstack/kayobe/+/850903 in Kayobe.

Reply:
This hasn't merged yet although the depending change has and Workflow+1 has been added. I am asking on IRC.

Reply:
This should be fixed by https://review.opendev.org/c/openstack/project-config/+/853003.

Reply:
https://github.com/stackhpc/kayobe/pull/39